EIPS/eip-2387.md

```diff
@@ -1,17 +1,17 @@
 ---
 eip: 2387
-title: "Hardfork Meta: Mountain Glacier"
+title: "Hardfork Meta: Muir Glacier"
 author: James Hancock (@madeoftin)
 discussions-to: https://ethereum-magicians.org/t/hard-fork-to-address-the-ice-age-eip-2387
 type: Meta
-status: Draft
+status: Last Call
```

Comment:
Please add `review-period-ends: 12-12-2019` (13th would be two weeks, but that's just Friday 13th :wink:)